tanksim.py

```diff
@@ -68,84 +69,77 @@ def __init__(self):
         if num > 900:
             brand = brands['BRO']
 
-        self.fermtime = brand['fermtime']
-        self.bYield = brand['bYield']
-        self.cYield = brand['cYield']
-        self.pYield = np.random.normal(94, 2, 1)
+        self.ferm_time = brand['ferm_time']
+        self.brew_yield = brand['brew_yield']
+        self.cellar_yield = brand['cellar_yield']
+        self.packaging_yield = np.random.normal(94, 2, 1)
 
 def generate_tanks(count_240, count_90):
     """Given numbers of 240 barrel (3 turn) and 90 barrel (1 turn) tanks, generate em"""
-    tanks = []
-    for i in range(count_240):
-        tanks.append(Tank('(240)FV ' + str(i), 3))
-    for i in range(count_90):
-        tanks.append(Tank('(90)FV ' + str(i), 1))
-    return tanks
-
-
-def emptyTanks():
-    return [tank.name for tank in tanks if not tank.is_filled()]
+    return ([Tank('(240)FV ' + str(i), 3) for i in range(count_240)] + 
+            [Tank('(90)FV ' + str(i), 1) for i in range(count_90)])
 
 def simulation(start,end,tanks,maxturns,daysoff):
-
-  d = start
-  delta = datetime.timedelta(days=1)
-  # first turn -- a turn is a brewhouse unit of work. brewhouses can do some max
-  # num of these per day.
-  turn = 1
-
-  weekSums = []
-  
-
-  f = open('testCSV.csv', 'w+')
-  writer = csv.writer(f)
-  writer.writerow( ('Tank', 'Brand', 'Date', 'Turn Bucket') )
-
-
-  while d <= end:
-
-    for tank in tanks:
-        
-        emptyList = emptyTanks()
-
-        while tank.is_filled() == False and d.weekday() not in daysoff and turn <= maxturns:
-            tank.num_brews_in_tank += 1
-            if tank.num_brews_in_tank == 1:
-                # fill tank with random brand
-                tank.update_fill()
-                #print "Tank ", tank.name, " filled with ", tank.brand, tank.brand.fermtime
-
-            writer.writerow( (tank.name, tank.brand, d.strftime("%a %Y-%m-%d"), turn) )
-            turn += 1
-        
-        # to prevent loop getting stuck on daysoff
-        if len(emptyList) == 0 or d.weekday() in daysoff:
-            if turn <= maxturns:
-                writer.writerow( ("EMPTY", "NULL", d.strftime("%a %Y-%m-%d"), turn) )
-            turn += 1
-
-        # ... 
-        if turn > 3:
-            turn = 1
-            d += delta
-            for tank in tanks:
-                tank.day_tick()
-                tank.reset_tank_if_complete()
-        
-            # Sunday
-            if d.weekday() == 6:
-                bblSum = sum([tank.totalBBLs for tank in tanks])
-                if bblSum > 0:
-                    weekSums.append(bblSum) 
-                #print "total BBLs at end of week: ", bblSum 
-                #print weekSums
-                weekTots = [t - s for s, t in zip(weekSums, weekSums[1:])]
-                #print weekTots
-
-  #print weekSums
-  print "Average Weekly Simulated BBLs: ", round(sum(weekTots)/len(weekTots), 2)
-  #print "total bbls for period:  ", bblSum
-  f.close()
+    d = start
```

Comment:
Most all the changes here are just whitespace, the indentation on the left was goofed